extensions/sftp/processors/ListSFTP.cpp

```diff
@@ -359,21 +360,21 @@ bool ListSFTP::filterFile(const std::string& parent_path, const std::string& fil
 
   /* Age */
   time_t now = time(nullptr);
-  uint64_t file_age = (now - attrs.mtime) * 1000;
+  std::chrono::milliseconds file_age = std::chrono::seconds(now - attrs.mtime);
   if (file_age < minimum_file_age_) {
     logger_->log_debug("Ignoring \"%s/%s\" because it is younger than the Minimum File Age: %ld ms < %lu ms",
         parent_path.c_str(),
         filename.c_str(),
-        file_age,
-        minimum_file_age_);
+        file_age.count(),
```

Comment:
`%ld` might not be the right one here (it seems it never was)

Reply:
changed all debug logs to use the `PRId64` and `static_cast<int64_t>` in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/78f8658fde802c73a5965dd734a544acbc37dca3 and https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/1fcbcc05ce27c9f7386c8f5b3a4a5267593c8fa1